src/promptflow/promptflow/_cli/_pf/_tool.py

```diff
@@ -98,11 +99,42 @@ def add_parser_list_tool(subparsers):
     )
 
 
+def add_parser_validate_tool(subparsers):
+    """Add tool list parser to the pf tool subparsers."""
+    epilog = """
+Examples:
+
+# Validate single function tool:
+pf tool validate -–source <package_name>.<module_name>.<tool_function>
+# Validate all tool in a package tool:
+pf tool validate -–source <package_name>
+# Validate tools in a python script:
+pf tool validate --source <path_to_tool_script>
+"""  # noqa: E501
+
+    def add_param_source(parser):
+        parser.add_argument("--source", type=str, help="The tool source to be used.", required=True)
+
+    return activate_action(
+        name="validate",
+        description="Validate tool.",
+        epilog=epilog,
+        add_params=[
+            add_param_source,
+        ],
+        subparsers=subparsers,
+        help_message="Validate tool. Will raise error if it is not valid.",
```

Comment:
Will raise error or return error?

Reply:
If an invalid tool is included, it will exit 1